x-pack/plugins/observability_solution/apm/public/components/shared/stacktrace/cause_stacktrace.tsx

```diff
@@ -24,7 +24,7 @@ const CausedByContainer = styled('h5')`
 const CausedByHeading = styled('span')`
   color: ${({ theme }) => theme.euiTheme.colors.textSubdued};
   display: block;
-  font-size: ${({ theme }) => theme.euiTheme.size.xs};
+  font-size: ${({ theme }) => useEuiFontSize('xs').fontSize};
```

Comment:
No need for `theme` argument in the callback. Applies to other places.

```suggestion
  font-size: ${() => useEuiFontSize('xs').fontSize};
```